resources/dicts/patrols/mountainous/border/any.json

```diff
@@ -639,5 +755,249 @@
+      "exp": 0,
+      "weight": 10,
+      "dead_cats": ["r_c"],
+      "history_text": {
+        "reg_death": "This cat died to keep c_n safe from a wolf.",
+        "lead_death": "died to keep c_n safe from a wolf"
+      },
+      "relationships": [
+        {
+          "cats_to": ["r_c"],
+          "cats_from": ["clan"],
+          "mutual": false,
+          "values": ["trust", "respect"],
+          "amount": 15
+        }
+      ]
+    },
+    {
+      "text": "r_c would give their life to protect c_n, and as {PRONOUN/r_c/subject} {VERB/r_c/charge/charges} past the huge wolf to lead it far, far from c_n, {PRONOUN/r_c/subject} {VERB/r_c/accept/accepts} it. Fear gives {PRONOUN/r_c/object} speed, but it only takes one successful hit. Miraculously {PRONOUN/r_c/subject} {VERB/r_c/fly/flies} out of the dog's jaws while it shakes {PRONOUN/r_c/object}, landing in a refuge. r_c is injured, desperate, and terrified - but {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} alive.",
+      "exp": 0,
+      "weight": 10,
+      "injury": [
+        {
+          "cats": ["r_c"],
+          "injuries": ["big_bite_injury"],
+          "scars": ["BRIGHTHEART"]
+        }
+      ],
+      "history_text": {
+        "scar": "This cat is scarred from bravely defending c_n against a wolf.",
+        "reg_death": "This cat died to keep c_n safe from a large wolf.",
+        "lead_death": "died to keep c_n safe from a large wolf"
+      },
+      "relationships": [
+        {
+          "cats_to": ["r_c"],
+          "cats_from": ["clan"],
+          "mutual": false,
+          "values": ["trust", "respect"],
+          "amount": 30
+        }
+      ]
+    },
+    {
+      "text": "s_c knows wolves are dangerous, but {PRONOUN/s_c/subject} {VERB/s_c/are/is} stronger and better than a simple beast. However, s_c underestimates the dog's power. In the struggle, the wolf manages to overpower {PRONOUN/s_c/object} and delivers a vicious bite, leaving {PRONOUN/s_c/object} with a deep wound on {PRONOUN/s_c/poss} leg. {PRONOUN/s_c/subject/CAP} narrowly escapes with {PRONOUN/s_c/poss} life.",
+      "exp": 0,
+      "weight": 20,
+      "stat_trait": ["ambitious", "arrogant"],
+      "injury": [
+        {
+          "cats": ["s_c"],
+          "injuries": ["mangled_leg"],
+          "scars": ["MANLEG"]
+        }
+      ],
+      "history_text": {
+        "scar": "This cat bears a mangled leg from a confrontation with a large wolf.",
+        "reg_death": "This cat died from a confrontation with a large wolf.",
+        "lead_death": "lost a life to a large wolf"
```

Comment:
don't remember if these are old events or new ones, but they shouldn't say "lost a life", it'll sound awkward when combined with the history text.  should say "died" instead.

Reply:
Old event, but fixed now!